CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Object Server API Changes (In Beta)
 
 * Add a default `UserStore` based on the Realm Object Store (`ObjectStoreUserStore`).
+* Add 'like' predecate for String fields (3752)
```

Comment:
We prefix issues with `#` so `#3752`. 

Perhaps rephrase to "Add `like` predicate for String fields to `RealmQuery` (#3752)` to make it more clear?